floodwatch/src/js/components/Chart.js

```diff
@@ -0,0 +1,270 @@
+// @flow
+
+import React, { Component } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import * as d3 from 'd3';
+import _ from 'lodash';
+import colors from './colors'
+import type {StackedData} from './FilterParent'
+
+type PropsType = {
+  barData: Array<Array<StackedData>>,
+  currentTopic: string,
+  side: string,
+  updateMouseOver: (topic: string) => void
+};
+
+type StateType = {
+  height: number,
+  svg: Object
+};
+
+function initialState(): Object {
+  return {
+    height:500
```

Comment:
This should return type `StateType` - which would need to include an `svg` key. Since it won't be set until `componentDidMount()` then it should be an optional property.